src/xcode/ENA/ENA.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,34 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "FMDB",
```

Comment:
why does this pull request add new packages?

Reply:
no new dependencies. We checked in the Package.resolved file